types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -9,7 +14,7 @@ interface UseChatUtilsHook {
   goToMoveChat: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `goToContactScreen` now requires a parameter of type `ContactScreenPayload`. This is a breaking change if there are any existing calls to this function that do not provide this parameter. Please ensure all calls to this function have been updated accordingly.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->